lib/rides/widgets/search_suggestion_filter.dart

```diff
@@ -0,0 +1,449 @@
+        _minComfortRating == _defaultRating &&
+        _minSafetyRating == _defaultRating &&
+        _minReliabilityRating == _defaultRating &&
+        _minHospitalityRating == _defaultRating;
+    bool isFeaturesDefault = _selectedFeatures.equals(_defaultFeatures);
+    bool isDeviationDefault = _maxDeviationController.text == _defaultDeviation;
+
+    List<Widget> widgets = [];
+    if (!isRatingDefault) {
+      List<Widget> ratingWidgets = [];
+      if (_minRating != _defaultRating) {
+        ratingWidgets.add(_buildSmallRatingIndicator(_minRating));
+        if (_minComfortRating != _defaultRating ||
+            _minSafetyRating != _defaultRating ||
+            _minReliabilityRating != _defaultRating ||
+            _minHospitalityRating != _defaultRating) {
+          ratingWidgets.add(const SizedBox(width: 4));
+        }
+      }
+      if (_minComfortRating != _defaultRating) {
+        ratingWidgets.add(
+          _buildSmallRatingIndicator(
+            _minComfortRating,
+            icon: Icon(
+              Icons.chair,
+              color: Theme.of(context).colorScheme.primary,
+            ),
+          ),
+        );
+      }
+      if (_minSafetyRating != _defaultRating) {
+        ratingWidgets.add(
+          _buildSmallRatingIndicator(
+            _minSafetyRating,
+            icon: Icon(
+              Icons.health_and_safety,
+              color: Theme.of(context).colorScheme.primary,
+            ),
+          ),
+        );
+      }
+      if (_minReliabilityRating != _defaultRating) {
+        ratingWidgets.add(
+          _buildSmallRatingIndicator(
+            _minReliabilityRating,
+            icon: Icon(
+              Icons.timer,
+              color: Theme.of(context).colorScheme.primary,
+            ),
+          ),
+        );
+      }
+      if (_minHospitalityRating != _defaultRating) {
+        ratingWidgets.add(
+          _buildSmallRatingIndicator(
+            _minHospitalityRating,
+            icon: Icon(
+              Icons.favorite,
+              color: Theme.of(context).colorScheme.primary,
+            ),
+          ),
+        );
+      }
+      int numDividers = ratingWidgets.length - 1;
+      for (int i = 0; i < numDividers; i++) {
+        ratingWidgets.insert(i * 2 + 1, const SizedBox(width: 10));
+      }
+      Widget ratingsRow = Row(children: ratingWidgets);
+      widgets.add(ratingsRow);
+    }
+    if (!isFeaturesDefault) {
+      Widget featuresRow = Row(children: _selectedFeatures.map((feature) => feature.getIcon(context)).toList());
+      widgets.add(featuresRow);
+    }
+    if (!isDeviationDefault) {
+      Widget deviationWidget = Row(
+        children: [const Icon(Icons.schedule), const SizedBox(width: 6), Text("± ${_maxDeviationController.text}")],
+      );
+      widgets.add(deviationWidget);
+    }
+    Widget sortingWidget = Row(
+      children: [const Icon(Icons.sort), Text(_sorting.getDescription(context))],
```

Comment:
can we move this part always to the right side

Reply:
You mean the entire row other than the filter icon should be aligned to the right or only the sort section or am I misunderstanding you entirely?

Reply:
Pretty sure he means just the sort section. I agree that it would look better!

Reply:
So it would not be scrollable anymore and glued to the right like the filter icon is glued to the left? I think that would be a bad decision because then the filter icon would become confusing because they look like they should be doing the same, but one indicates what the entire row is for and the other indicates a filter.

Reply:
Okay but it's the only thing that is always shown. And right now, if there are no other filters, it kind of floats weirdly in the middle (but not symmetrically) of the screen

Reply:
I can align the whole thing to the right?

Reply:
Did that